tests/common/reboot.py

```diff
@@ -241,6 +241,10 @@ def reboot(duthost, localhost, reboot_type='cold', delay=10,
     except KeyError:
         raise ValueError('invalid reboot type: "{} for {}"'.format(reboot_type, hostname))
 
+    # Create a temporary file in tmpfs before reboot
+    logger.info('DUT {} create a file /dev/shm/test_reboot before rebooting'.format(hostname))
+    duthost.command('sudo touch /dev/shm/test_reboot')
```

Comment:
Minor: it may be a good idea to verify that this `touch` command was successful. If the FS is readonly, the issue can be caught right away?

Reply:
https://www.kernel.org/doc/Documentation/filesystems/tmpfs.txt - As per this kernel documentation, /dev/shm will be always mounted for 2.3 and above kernel versions and this /dev/shm is always mounted as a read-write filesystem in memory.